frontend/src/components/Notificacao/Form/ExamesImagem/TomografiaTorax.vue

```diff
@@ -81,6 +87,15 @@ export default {
     updateTomografiaOutro(tomografiaOutro) {
       this.$emit('update:tomografiaOutro', tomografiaOutro);
     },
+    requiredIfRealizouOutraTomografiaTorax(value) {
+      if (!this.realizouOutraTomografiaTorax) {
+        return true;
+      }
+      return required(value, 'O campo é obrigatório.');
```

Comment:
Mesma coisa, pode remover o parâmetro
```suggestion
      return required(value);
```